data/profiles.json

```diff
@@ -185,5 +185,21 @@
       "https://www.youtube.com/channel/UCmggGelTqiq2L3q5BNAKx9w",
       "https://www.uiug.in"
     ]
+  },

```

Comment:
This doesn’t look to be in the right alphabetical position

Reply:
I think that's correct now? 😅

Reply:
Perfect 👌🏻